docs/storybook-react/.storybook/main.ts

```diff
@@ -0,0 +1,50 @@
+import type { StorybookConfig } from '@storybook/react-vite';
+
+import path, { join, dirname } from 'path';
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that are set up within a monorepo.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function getAbsolutePath(value: string): any {
+  return dirname(require.resolve(join(value, 'package.json')));
+}
+const config: StorybookConfig = {
+  stories: [
+    '../src/**/*.mdx',
+    '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)',
+    // '../../../packages/react-components/src/**/*.stories.@(js|jsx|ts|tsx)', TO DO: Update when react component package is created
```

Comment:
Will update when package is created and first story is working (component template)